src/common/components/storage/StorageDetail.tsx

```diff
@@ -68,12 +100,74 @@ const DisksTable = ({
   onDiskRole,
 }: DisksTableProps) => {
   const isEditable = !!canEditDisks?.(host);
+  const dispatch = useDispatch();
+  const { addAlert } = useAlerts();
+  const { data: cluster } = useSelector(selectCurrentClusterState);
+  const updateDiskSkipFormatting = React.useCallback(
+    async (checked: boolean, hostId: Host['id'], diskId: Disk['id']) => {
+      try {
+        if (!cluster) {
+          return new Error(`Cannot update disksSkipFormatting in host ${hostId}\nMissing cluster`);
+        }
+        if (!diskId) {
+          // noinspection ExceptionCaughtLocallyJS
```

Comment:
We should not add inspections for a specific IDE.

Reply:
Copy and paste typo